WebRoot/public/extuserprov.jsp

```diff
@@ -28,6 +28,12 @@
 
 <c:set var="client" value="${param.client}"/>
 <c:set var="version" value="${initParam.zimbraCacheBusterVersion}"/>
+<c:if test="${not empty errorCode}">
+	<c:set var="validationErrorCode" value="${errorCode}" />
+</c:if>
+<c:if test="${not empty header['errorCode']}">
+	<c:set var="validationErrorCode" value="${header['errorCode']}" />
+</c:if>
```

Comment:
We can shorthand this as below:
 <c:set var="validationErrorCode" value="${not empty errorCode?errorCode:(not empty header['errorCode'])?header['errorCode']:null />